clamav/clamav.lua

```
@@ -1,318 +1,297 @@
-	end
-	-- Receive response
-	local data
-	data, err = clamav_socket:receive("*l")
-	if not data then
-		clamav_socket:close()
-		return false, err
-	end
-	clamav_socket:close()
-	return true, data
+-- Improved filename extraction function
+function clamav:extract_filename(headers)
```

Luacheck (1.2.0)

[warning] unused argument 'self'

(W212)